charts/aws-event-sources/0.1.0/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: aws-event-sources
+version: 0.1.0
+appVersion: 0.2.0
+type: application
+description: A Helm chart for AWS event sources by Triggermesh
+home: https://github.com/triggermesh/aws-event-sources
+icon: https://raw.githubusercontent.com/triggermesh/aws-event-sources/master/images/klass-logo.png
```

Comment:
Please make a local copy of the logo in the parent folder `aws-event-sources/` and reference the image here as `file://../klass-logo.png`. This way the logo can be displayed in airtight environments.